src/poliastro/tests/test_plotting2d.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from unittest import mock
+
+from astropy import units as u
+
+from poliastro.examples import iss
+from poliastro.plotting import OrbitPlotter2D
+
+
+def test_get_figure_has_expected_properties():
+    frame = OrbitPlotter2D()
+    figure = frame.figure
+
+    assert figure["data"] == [{}]
+    assert figure["layout"]["autosize"] is True
+    assert "xaxis" in figure["layout"]["scene"]
+    assert "yaxis" in figure["layout"]["scene"]
+    assert "aspectmode" in figure["layout"]["scene"]
+
+
+def test_set_different_attractor_raises_error():
+    body1 = mock.MagicMock()
```

Comment:
Do we need Mocks here? Can't we use the `Earth` and `Mars` for instance?

Reply:
Noted!